core/experiments/manager.php

```diff
@@ -149,6 +152,44 @@ public function add_feature( array $options ) {
 		return $experimental_data;
 	}
 
+	/**
+	 * Structurize the feature tags.
+	 *
+	 * @param string $tag
+	 * @param array $tags
+	 *
+	 * @return array
+	 */
+	private function format_feature_tags( string $tag, array $tags ) {
+		if ( ! empty( $tag ) ) {
+			$tags[] = [
+				'type' => 'default',
+				'label' => $tag,
+			];
+		}
+
+		if ( empty( $tags ) ) {
+			return $tags;
+		}
+
+		$allowed_tag_properties = [ 'type', 'label' ];
+
+		$default_tag = [
+			'type' => 'default',
+			'label' => '',
+		];
+
+		foreach ( $tags as &$tag ) {
+			if ( empty( $tag['label'] ) ) {
+				unset( $tag );
+			}
+
+			$tag = $this->merge_properties( $default_tag, $tag, $allowed_tag_properties );
+		}
+
+		return $tags;
```

Comment:
Please add it to the usage data + schema

Reply:
Added